src/main/java/seedu/trackbeau/model/booking/BookingDateTime.java

```diff
@@ -0,0 +1,57 @@
+package seedu.trackbeau.model.booking;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.trackbeau.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class BookingDateTime {
+    public static final String MESSAGE_CONSTRAINTS =
+            "BookingDateTime should follow dd-mm-yyyy hh:mm, and it should not be blank";
```

Comment:
I think the datetime format should be consistent with the actual one i.e. dd-MM-yyyy HH:mm